config/ext_jenkins-release_privatek8s.yaml

```diff
@@ -0,0 +1,692 @@
+serviceAccount:
+  create: true
+  name: jenkins-controller
+# serviceAccountAgent:
+#   create: true
+#   name: jenkins-agent
+serviceAccountAgent:
+  create: false
+rbac:
+  create: true
+  readSecrets: true
+persistence:
+  enabled: true
+  size: 50Gi
+agent:
+  componentName: "agent"
+networkPolicy:
+  # As of today, 2020-06-19, network policy is not supported for windows node
+  # https://docs.microsoft.com/en-us/azure/aks/windows-node-limitations#are-all-features-supported-with-windows-nodes
+  # This is a blocker for the release environment as in the current state,
+  # windows containers are not allowed to connect on main Jenkins.
+  # I disable it for now
+  # enabled: true
+  enabled: false
+  internalAgents:
+    allowed: true
+    namespaceLabels:
+      name: "jenkins-release"
+controller:
+  image: jenkinsciinfra/jenkins-lts
+  tag: 0.28.0-2.387.1
+  imagePullPolicy: IfNotPresent
+  nodeSelector:
+    kubernetes.io/os: linux
+    kubernetes.azure.com/agentpool: linuxpool
+  resources:
+    limits:
+      cpu: "2"
+      memory: "4Gi"
+    requests:
+      cpu: "2"
+      memory: "4Gi"
+  probes:
+    startupProbe:
+      initialDelaySeconds: 120
+    livenessProbe:
+      initialDelaySeconds: 120
+    readinessProbe:
+      initialDelaySeconds: 120
+  testEnabled: false
+  podSecurityContextOverride:
+    runAsNonRoot: true
+    runAsUser: 1000
+    supplementalGroups: [1000]
```

Comment:
```suggestion
```
I wonder if we should or not remove this part, not present in the infra values.

WDYT?

Reply:
Removed as it prevented the init container to write on the PV